chart/templates/_helpers.yaml

```
@@ -731,85 +731,148 @@ server_tls_key_file = /etc/pgbouncer/server.key
 {{- end }}
 
 {{/*
-Set the default value for securityContext
-If no value is passed for securityContext or <node>.securityContext, defaults to global uid and gid.
+Set the default value for pod securityContext
+If no value is passed for securityContexts.pod or <node>.securityContexts.pod or legacy securityContext and <node>.securityContext, defaults to global uid and gid.
 
-    +------------------------+      +-----------------+      +-------------------------+
-    | <node>.securityContext |  ->  | securityContext |  ->  | Values.uid + Values.gid |
-    +------------------------+      +-----------------+      +-------------------------+
+    +-----------------------------+      +------------------------+      +----------------------+      +-----------------+      +-------------------------+
+    | <node>.securityContexts.pod |  ->  | <node>.securityContext |  ->  | securityContexts.pod |  ->  | securityContext |  ->  | Values.uid + Values.gid |
+    +-----------------------------+      +------------------------+      +----------------------+      +-----------------+      +-------------------------+
 
-Values are not accumulated meaning that if runAsUser is set to 10 in <node>.securityContext,
+Values are not accumulated meaning that if runAsUser is set to 10 in <node>.securityContexts.pod,
 any extra values set to securityContext or uid+gid will be ignored.
 
 The template can be called like so:
-   include "airflowSecurityContext" (list . .Values.webserver)
+   include "airflowPodSecurityContext" (list . .Values.webserver)
 
 Where `.` is the global variables scope and `.Values.webserver` the local variables scope for the webserver template.
 */}}
-{{- define "airflowSecurityContext" -}}
-  {{- $ := index . 0 }}
+{{- define "airflowPodSecurityContext" -}}
+  {{- $ := index . 0 -}}
   {{- with index . 1 }}
-    {{- if .securityContext }}
-      {{- toYaml .securityContext }}
-    {{- else if $.Values.securityContext }}
-      {{- toYaml $.Values.securityContext }}
-    {{- else }}
+    {{- if .securityContexts.pod -}}
+{{ toYaml .securityContexts.pod | print }}
```

```suggestion
      {{- toYaml .securityContexts.pod | print }}
```

Can you fix the indentation here?